Oracle/cdm_transform_tests_BACKUP_49608.sql

```diff
@@ -1,180 +0,0 @@
-/* Test to make sure we got about the same number of patients in the CDM 

```

Comment:
Gesundheit! I have certainly had my share of git-sneezes.

I expect it will be worth fixing this history if it affects anything we contribute upstream, but it's not a problem for now.